style.css

```diff
@@ -0,0 +1,31 @@
+/* make the navbar fixed to the top of the viewport */
+nav#navbar {
+    position: fixed;
+    height: 5vh;
+    top: 0;
+    width: 100%;
+    font-size: 35px;
+    background-color: rgb(32, 253, 3);
+  }
+  
+  /* make the welcome section the same height as the viewport */
+  section#welcome-section {
```

Comment:
I'd suggest you avoid leaving white spaces before classes. This will make your code cleaner